Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -309,6 +309,29 @@ QString GameFile::GetLanguage(DiscIO::Language lang) const
   }
 }
 
+QString GameFile::GetFileType() const
+{
+  switch (GetBlobType())
+  {
+  case DiscIO::BlobType::PLAIN:
+    return QObject::tr("Plain");
```

Comment:
I'd also thought about it but I'm not sure, as it would then also display "ISO" on non-games, like WADs. Maybe a separate check? (ie. if it isn't a disk-based game, display "WAD")